services/harmony/app/server.ts

```diff
@@ -20,19 +20,28 @@ import * as exampleBackend from '../example/http-backend';
 import cmrCollectionReader from './middleware/cmr-collection-reader';
 import * as fs from 'fs';
 
+// redact specific values from a request log
+export const requestFilter: expressWinston.RequestFilter = (req, propName) => {
+  if (propName === 'headers') {
+    const redacteProp = req[propName];
+    if (redacteProp.cookie) {
+      redacteProp.cookie = '<redacted>';
```

Comment:
nit: better to create a constant for 'redacted' since you are using it in more than one place